website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,259 @@
+
+
+* <a name="v-server" href="#v-server">`server`</a> - Values that configure
+  running a Consul server cluster within Kubernetes.
+
+  - <a name="v-server-enabled" href="#v-server-enabled">`enabled`</a> (`boolean: global.enabled`) -
+  If true, the chart will install all the resources necessary for a Consul
+  server cluster. If you're running Consul externally and want agents within
+  Kubernetes to join that cluster, this should probably be false.
+
+  - <a name="v-server-image" href="#v-server-image">`image`</a> (`string: global.image`) -
+  The name of the Docker image (including any tag) for the containers running
+  Consul server agents.
+
+  - <a name="v-server-replicas" href="#v-server-replicas">`replicas`</a> (`integer: 3`) -
+  The number of server agents to run. This determines the fault tolerance
+  of the cluster. Please see the [deployment table](/docs/internals/consensus.html#deployment-table)
+  for more information.
+
+  - <a name="v-server-bootstrapexpect" href="#v-server-bootstrapexpect">`bootstrapExpect`</a> (`integer: 3`) -
+  For new clusters, this is the number of servers to wait for before
+  performing the initial leader election and bootstrap of the cluster. This
+  must be less than or equal to `server.replicas`. This value is only
+  used when bootstrapping new clusters, it has no effect during ongoing cluster
+  maintenance.
+
+  - <a name="v-server-connect" href="#v-server-connect">`connect`</a> (`boolean: true`) -
+  This will enable/disable [Connect](/docs/connect/index.html). Setting this
+  to true _will not_ automatically secure pod communication, this setting will
+  only enable usage of the feature. Consul will automatically initialize a new
+  CA and set of certificates. Additional Connect settings can be configured
+  by setting the `server.extraConfig` value.
+
+  - <a name="v-server-disruptionbudget" href="#v-server-disruptionbudget">`disruptionBudget`</a> -
+  This configures the
+  [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+  for the server cluster.
+
+      * <a name="v-server-disruptionbudget-enabled" href="#v-server-disruptionbudget-enabled">`enabled`</a> (`boolean: true`) -
+      This will enable/disable registering a PodDisruptionBudget for
+      the server cluster. If this is enabled, it will only register the
+      budget so long as the server cluster is enabled.
+
+      * <a name="v-server-disruptionbudget-maxunavailable" href="#v-server-disruptionbudget-maxunavailable">`maxUnavailable`</a> (`integer: null`) -
+      The maximum number of unavailable pods. By default, this will be automatically
+      computed based on the `server.replicas` value to be `(n/2)-1`.
+
+  - <a name="v-server-extraconfig" href="#v-server-extraconfig">`extraConfig`</a> (`string: "{}"`) -
+  A raw string of extra JSON or HCL configuration for Consul servers. This
+  will be saved as-is into a ConfigMap that is read by the Consul agents.
```

Comment:
nit: This is technically correct but might be worth clarifying to "server agents" given that we all (and most Consul users) seem to use just "agents" to refer to "client agents".

Reply:
Ack done.